webapp/channels/src/components/suggestion/switch_channel_provider.tsx

```diff
@@ -408,7 +409,7 @@ function makeChannelSearchFilter(channelPrefix: string) {
                         continue;
                     }
                     const {nickname, username} = user;
-                    userString = `${nickname}${username}${Utils.getFullName(user)}`;
+                    userString = `${nickname}${SEPARATOR}${username}${SEPARATOR}${Utils.getFullName(user)}`;
```

Comment:
more clean
```suggestion
                    userString = [nickname, username, Utils.getFullName(user)].join(SEPARATOR);
```

Reply:
Thanks for the suggestion @tasawar-hussain